src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java

```diff
@@ -55,6 +55,16 @@ public class CosmosDBConfig extends AbstractConfig {
             "A comma delimited list of Kafka topics mapped to Cosmos containers.\n"
                     + "For example: topic1#con1,topic2#con2.";
 
+    public static final String COSMOS_GATEWAY_MODE_ENABLED = "connect.cosmos.connection.gateway.enabled";
+    private static final String COSMOS_GATEWAY_MODE_ENABLED_DOC =
+            "Flag to indicate whether to use gateway mode. By default it is false";
+    private static final boolean DEFAULT_COSMOS_GATEWAY_MODE_ENABLED = false;
+
+    public static final String COSMOS_CONNECTION_SHARING_ENABLED = "connect.cosmos.connection.sharing.enabled";
+    private static final String COSMOS_CONNECTION_SHARING_ENABLED_DOC =
```

Comment:
Worth mentioning more details about connection sharing. Also, usually connection sharing will happen for first client with other clients. So first client's connection settings will be used for other clients. 
If there is a way to set these up, we should mention these details in the doc.

Reply:
how many cosmosdb clients will be created will be eventually decided by max.tasks, topic partitions etc - so I think it kind makes sense to use the same config for each one. Or if in the future we find the needs to have different config between clients we can then decide how to support it.

updated the doc to contain a little bit more info.